libs/db/db-core/src/main/kotlin/net/corda/db/core/BaseDataSourceFactory.kt

```diff
@@ -2,17 +2,19 @@ package net.corda.db.core
 
 import java.time.Duration
 
-class PostgresDataSourceFactory(
-    private val datasourceFactory: DataSourceFactory = HikariDataSourceFactory()
-) {
+class BaseDataSourceFactory(
```

Comment:
```suggestion
class DataSourceFactory(
```
Not actually a base class anymore.